graphx/src/main/scala/org/apache/spark/graphx/lib/ShortestPaths.scala

```diff
@@ -25,7 +25,7 @@ import org.apache.spark.graphx._
  * Computes shortest paths to the given set of landmark vertices, returning a graph where each
  * vertex attribute is a map containing the shortest-path distance to each reachable landmark.
  */
-object ShortestPaths {
+object ShortestPaths extends Serializable {
```

Comment:
The two test changes aren't ultimately going to be necessary if the ClosureCleaner is fully updated, but, these are a few easy fixes to tests that can easily be changed to avoid some closure cleaning anyway.